Library/Formula/gcc.rb

```diff
@@ -21,9 +21,22 @@ def osmajor
 
   desc "GNU compiler collection"
   homepage "https://gcc.gnu.org"
-  url "http://ftpmirror.gnu.org/gcc/gcc-5.2.0/gcc-5.2.0.tar.bz2"
-  mirror "https://ftp.gnu.org/gnu/gcc/gcc-5.2.0/gcc-5.2.0.tar.bz2"
-  sha256 "5f835b04b5f7dd4f4d2dc96190ec1621b8d89f2dc6f638f9f8bc1b1014ba8cad"
+  head "https://github.com/gcc-mirror/gcc.git"
```

Comment:
Oh, I don't think I meant to commit the `head` line, and I definitely didn't test it.